packages/stores/src/providers/WebsiteStatusProvider.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { useSubscription } from '@deriv/api';
+import { observer } from 'mobx-react-lite';
+import useStore from '../useStore';
+import merge from 'lodash.merge';
+
+const WebsiteStatusProvider = observer(({ children }: React.PropsWithChildren<unknown>) => {
+    const { data, subscribe, unsubscribe } = useSubscription('website_status');
+    const {
+        client: { is_authorize },
+        website_status: { update },
+    } = useStore();
+
+    useEffect(() => {
+        if (data) {
+            const { website_status } = data;
+            if (website_status) update(prev => merge(prev, website_status));
+        }
+    }, [update, data]);
+
+    useEffect(() => {
```

Comment:
This useEffect handles the case where the response from website_status is different when client is logged in/out, we need to re-subscribe if client is logging in, can refer to socket-general.js for implementation reference